vpdq/python/tests/test_vpdq_hash.py

```diff
@@ -49,6 +49,31 @@ def test_vpdq_utils():
         assert vpdq.hash_to_hex(feature.hash) == hex_hash
 
 
+def test_error_checking():
+    video_file = Path(f"{video_folder}/{TEST_FILES[0]}.mp4")
+    with pytest.raises(ValueError) as error_info:
+        assert not vpdq.computeHash(
+            input_video_filename=video_file, seconds_per_hash=-1
+        )
```

Comment:
blocking: Remove these asserts - if you expect these files to throw errors, your raises() calls with throw if they don't throw the expected exception (try it out).

As is, they are unreachable/dead code, and so might confuse a reader